src/main/java/Duke.java

```diff
@@ -1,10 +1,61 @@
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
 public class Duke {
+    public static void printList(List<Task> list) {
+        for (int i = 0; i < list.size(); i++) {
+            System.out.println((i+1)+"."+list.get(i).getDoneStatus()+" "+list.get(i).getDescription());
```

Comment:
Insert spaces between your operators (in this case, insert spaces between your + to separate your operands).